src/games/strategy/engine/framework/map/download/DownloadFileParser.java

```diff
@@ -30,6 +44,7 @@
         private String description;
         private String mapName;
         private Version version;
+        private DownloadFileDescription.DownloadType downloadType;
```

Comment:
Not a package. `DownloadType` is a local static class attached  to DownloadFileDescription.